optimum/habana/__init__.py

```diff
@@ -23,6 +23,11 @@
     GaudiTrainer,
     GaudiTrainingArguments,
 )
+from .sentence_transformers import (
```

Comment:
This causes all tests to fail unless sentence-transformers are installed. Hence no tests can be run right now as we see `ModuleNotFoundError: No module named 'sentence_transformers'`, I think you have to update `setup.py` with the requirement. @libinta , @mandy-li  if there's a better fix, please comment here.

Reply:
Hi @vidyasiv i updated the setup.py with the addition of sentence transformers' package. Also passed with the test of tests/test_sentence_transformer.py and tests/sentence_transformers. Please check. Thanks.